packages/starship/build.sh

```
@@ -0,0 +1,30 @@
+# build.ncl — no explicit `cd` needed (mirrors delta/codex pattern).
+
+export CC=gcc
+export LD=gcc
+# `--remap-path-prefix` strips absolute build-time paths from the
+# binary so reruns are byte-identical regardless of build directory
+# or `$HOME`. Two mappings: source dir (where the tarball extracted)
+# and the cargo registry (where transitive dependency source lives).
+# `CARGO_INCREMENTAL=0` disables incremental compilation, which would
+# otherwise cache build state in non-deterministic ways. Both per
+# gominimal/minimal-repro's reproducibility-fixes guide.
+export RUSTFLAGS="-C linker=gcc --remap-path-prefix=$(pwd)=/builddir --remap-path-prefix=$HOME/.cargo=/cargo"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)